tests/end_to_end/features/steps/client.py

```diff
@@ -4,3 +4,26 @@
 @given("Featureform is installed")
```

Comment:
would it be possible to target a particular version?

Reply:
We could, but this is mainly to test against whatever is on the users current branch